### PR TITLE
Update Lisk SDK to v6.0.0-beta.0

### LINKED ISSUE #673: Update Lisk SDK to v6.0.0-beta.0
### Description

Update Lisk SDK to v6.0.0-beta.0.

### Acceptance Criteria

- Update SDK to v6.0.0-beta.0
- Update upstream interface changes
- Ensure Lisk Core builds and is able to start

### PR DESCRIPTION
**Pull request by ManuGowda**
### What was the problem?

This PR resolves #673 and #615

### How was it solved?

- :arrow_up: Bump version
- Updated config and genesis block for betanet

### How was it tested?

- Manually running core with network betanet
- Deployed network
- Made cross-chain transfers successfully

### CODE REVIEW

File: config/betanet/config.json [hunk 1]
```diff
@@ -20,9 +21,7 @@
 		"maxTransactionsSize": 15360
 	},
 	"generator": {
-		"keys": {
-			"fromFile": "./config/dev-validators.json"
-		}
+		"keys": {}
 	},
```
Comment (sameersubudhi) [inline config/betanet/config.json:25]: Optional: The entire `generator` property can probably be removed.

**Review by sameersubudhi — CHANGES_REQUESTED** [review]
We need to add few more accounts to the `genesis_assets.json`. @nagdahimanshu can assist you with it.

Also, please consider updating the config/genesis block/dependencies for devnet and alphanet as well. If we do not intend to use it anymore, then maybe entirely remove them.

File: config/betanet/config.json (hunk 1 above)
Comment (ManuGowda) [reply config/betanet/config.json:25]: @ishantiw and @shuse2 suggested to keep it, I tested the app and its working fine with this config. Let me know what you guys think.

Comment (shuse2) [reply config/betanet/config.json:25]: if it works, either ways are fine

Comment (sameersubudhi) [reply config/betanet/config.json:25]: It should work as is, hence the optional recommendation. 🙂

**Review by sameersubudhi — APPROVED** [review]
LGTM 👍

**Comment by nagdahimanshu** [comment]
@ManuGowda, @sameersubudhi `keepEventsForHeights` in `betanet` config is set to 300. Do we have to change it to -1?

**Review by shuse2 — APPROVED** [review]

**Review by sameersubudhi — APPROVED** [review]

**Pull request merged**